modules/text/include/opencv2/text/ocr.hpp

```diff
@@ -227,6 +235,11 @@ class CV_EXPORTS OCRHMMDecoder : public BaseOCR
                      std::vector<std::string>* component_texts=NULL, std::vector<float>* component_confidences=NULL,
                      int component_level=0);
 
+    // aliases for scripting
+    CV_WRAP cv::String run(Mat& image, int min_confidence, int component_level=0);
```

Comment:
same, please, use InputArray. BTW, cv:: is not needed, because you are inside cv namespace anyway